src/Scheduler/ScheduledCommandsResolver.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Scheduler;
+
+use DateTime;
+use Tempest\Console\ConsoleCommand;
+
+interface ScheduledCommandsResolver
```

Comment:
As mentioned further down below, I don't think this interface should exist, and the code should be moved as private methods to `GenericScheduler`

Reply:
We can def drop it if you want to - I've added it because you've mentioned that we should probably provide the user with option to do stateful or stateless scheduler dispatches - with this interface we won't have to change the `GenericScheduler` regardless of how the user may want to resolve commands to run, which seems like a nice thing to have tbh.

If we just want to go stateless in this PR without anticipating stateful to be implemented relatively soon, or at all, then it makes complete sense to drop it.

Lemme know your thoughts!

Reply:
Okay I missed your other comment - will remove